tools/gn

```diff
@@ -314,6 +314,33 @@ def setup_goma(args):
   return goma_gn_args
 
 
+# Find the locations of the macOS and iOS SDKs under flutter/prebuilts.
+def setup_apple_sdks():
+  sdks_gn_args = {}
+
+  # These are needed on a macOS host regardless of target.
+  # This value should agree with the 'mac_sdk_min' value in the DEPS file.
+  sdks_gn_args['mac_sdk_min'] = '10.14'
+  sdks_gn_args['mac_deployment_target'] = '10.14.0'
```

Comment:
If this is now the expected spot to update the versions, can you copy over the comment from `mac_sdk.gni`?
```
  # The MACOSX_DEPLOYMENT_TARGET variable used when compiling.
  # Must be of the form x.x.x for Info.plist files.
```
https://github.com/flutter/buildroot/blob/ebc2748229c8377d97ee1cafd11dff07c285f440/build/config/mac/mac_sdk.gni#L12-L13

Reply:
Done.